### PR TITLE
Fix/targeted content margins

### PR DESCRIPTION
**Pull request by davidsauntson**
Adds targeted content components to the list of things that give subsequent headings the correct margin.

### CODE REVIEW

**Review by baseonmars — APPROVED** [review]
Space to breath. so nice :+1:

File: scss/2-tools/_typography.scss [hunk 1]
```diff
@@ -28,7 +28,8 @@
   .cads-prose img + &,
   .cads-prose details + &,
```
Comment (davidrapson) [inline scss/2-tools/_typography.scss:29]: Can we remove `.cads-prose details` at the same time (that's the _old_ targeted content)

**Review by davidrapson — APPROVED** [review]
👍 with a little tidy up request

File: scss/2-tools/_typography.scss (hunk 1 above)
Comment (davidsauntson) [reply scss/2-tools/_typography.scss:29]: nice catch!

**Pull request merged**